internal/common/testing/container/container.go

```diff
@@ -79,7 +80,7 @@ func New(t *testing.T) *Containers {
 		t.Fatalf("failed creating docker client from environment: %v", err)
 	}
 
-	ctx := &Containers{cli: cli, t: t}
+	ctx := &Containers{cli: cli, runningContainers: make(map[ID]Container), t: t}
```

Comment:
Does "Containers" need to be public?

Reply:
again, not introduced in this PR but our golangci-lint suggests it should be:
```
exported func New returns unexported type *github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/testing/container.containers, which can be annoying to use
```